src/js/application.js

```diff
@@ -464,4 +494,6 @@ class Game {
   }
 }
 
-new Game(570, 550);
+const currentGame = new Game(570, 550);
+const undoButton = document.getElementsByClassName('undoButton')[0]
```

Comment:
I understand that this might not be the best way to implement this feature. I couldn't decide between adding a separate html button element as I have done right now (it seems semantically more appropriate) or adding a rectangle to the svg that acts like a button to undo last move (that way I wouldn't have to access the `undoLastMove` function from outside the class.
Which one of these would you say is better (or is there another way of solving this?)